pkg/subctl/cmd/join.go

```diff
@@ -6,13 +6,27 @@ import (
 	"github.com/spf13/cobra"
 
 	"github.com/submariner-io/submariner-operator/pkg/subctl/datafile"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/operator/deploy"
 	"github.com/submariner-io/submariner-operator/pkg/subctl/operator/install"
 )
 
+var (
+	clusterID   string
+	serviceCIDR string
+	clusterCIDR string
+)
+
 func init() {
 	rootCmd.AddCommand(joinCmd)
+	joinCmd.Flags().StringVar(&clusterID, "clusterid", "", "cluster ID used to identify the tunnels")
+	joinCmd.Flags().StringVar(&serviceCIDR, "servicecidr", "", "service CIDR")
```

Comment:
I will add the network detection on top to avoid the cmdline flags :)

Reply:
Yes, I was hoping you’d do that ;-). We’ll probably need the flags anyway, for case where the detection doesn’t find the values.

Reply:
True, I was thinking about:
0) Provided flags override anything, with a warning if they don't match detection
1) Detection -> works, use that
2) If there's a parameter missing we ask the admin, like for the gateway node.